### PR TITLE
use official coverage action

### PR DESCRIPTION
**Pull request by kalmarek**
fixes #231

### CODE REVIEW

File: .github/workflows/runtests.yml [hunk 1]
```diff
@@ -56,6 +56,11 @@ jobs:
...
-        env:
-          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+      - uses: julia-actions/julia-processcoverage@v1
+      - uses: codecov/codecov-action@v1
+        with:
+          file: ./lcov.info
+          flags: unittests
+          name: codecov-umbrella
+          fail_ci_if_error: false
+          token: ${{ secrets.CODECOV_TOKEN }}
```
Comment (benlorenz) [inline .github/workflows/runtests.yml:66]:
```suggestion
```
can we try without that token? It should not be necessary and we don't have that in our repository settings.

**Comment by kalmarek** [comment]
@benlorenz have you seen the crash on 1.4, ubuntu-latest ?

**Comment by benlorenz** [comment]
No haven't seen that one yet, will be gone with the next retry I guess.... And no way to debug this...
I like that there is no mention of neither polymake nor cxxwrap in the bracktrace.

But coverage seems to work without the token.

Forcing 0.9.1 blocks the branch-protection rules, shall I change them. Using 0.9 also pulls in 0.9.1.

**Comment by benlorenz** [comment]
I'm seeing a reproducible crash during testing with julia 1.4.1 with my gentoo-julia, maybe this is related. (and maybe specific to 1.4.1 which github is also using now)

I will test with the julia binaries and then open a new ticket depending on the results.

**Comment by kalmarek** [comment]
I removed forcing `0.9.1`, thanks for noticing, this was committed by accident;

I'm running the official 1.4.1 locally (no libraries override) and can not reproduce it, unfortunately;

**Comment by kalmarek** [comment]
and for some reason the crash is gone ;) please bless and merge when appropriate

**Pull request merged**